.github/workflows/kind.yml

```diff
@@ -102,6 +102,11 @@ jobs:
         mkdir log
         mkdir test-e2e-encap-coverage
         ANTREA_LOG_DIR=$PWD/log ANTREA_COV_DIR=$PWD/test-e2e-encap-coverage ./ci/kind/test-e2e-kind.sh --encap-mode encap --coverage
+    - name: Generate coverage files
+      run: |
+        cd test-e2e-encap-coverage
+        for dir in */; do go tool covdata textfmt -i="${dir}" -o "${dir%?}_$(date +%Y-%m-%d_%H-%M-%S).cov.out" && rm -rf "${dir}"; done
+        cd -
```

Comment:
Looking at this, I am assuming that this conversion is necessary and that the files which are generated by default (`covcounters.X`, `covemeta.X`) are not understood natively by codecov?

I think that this conversion should happen before that (maybe in `test-e2e-kind.sh`?) so that we don't have to repeat this multiple times for every test job.